lib/Entity/Playlist.php

```diff
@@ -928,4 +931,56 @@ public function cloneClosureTable($newParentId)
             'parentId' => $this->playlistId
         ]);
     }
+
+    public function generatePlaylistMapping($widgets, $parentId, &$playlistMappings, &$count, &$nestedPlaylistDefinitions, &$dataSetIds, &$dataSets, $dataSetFactory, $includeData)
```

Comment:
PHP Doc would be good here to explain what this does and why we're passing by reference